### DISCUSSION TITLE
Sveltekit error: missing /auth/csrf route

### DISCUSSION THREAD

**Discussion by socketopp** [nextauthjs/next-auth#9520] [Help]
Just curious, I have just setup a project and trying to deploy it to cloudflare using the sveltekit framework.

Keep getting `"Error: Not found: /auth/csrf"` when I am trying to sign in with Github. The request made is GET to 
`https://[domain].com/auth/csrf`

What plausible errors could there be if it cannot find the auth/csrf route? 

I've added the NEXTAUTH_URL to the base url of the website I am trying to login on as an environmental variable.

**Comment by akubima** [comment]
I have the same issue.

**Reply by socketopp** [reply]
@peprmint1 
My root cause was using Resend inside one of my adapters. I made a ticket on [Resends nodejs-libary](
https://github.com/resend/resend-node/issues/289).  I resolved my issue by using Resends REST API instead.

I was using it in one my custom provider for sending email. I would advise you to overlook your providers, add/remove provider and see if the issue remain.

Good luck on debugging.

**Comment by socketopp** [comment]
@peppermint01 
Did you manage to fix your issue? I noticed it occuring in SvelteKit v.2

**Comment by qudo-code** [comment]
has this been deprecated? also noticing it works on our v1 app but not v2.
